src/test/java/core/basesyntax/strategy/impl/TransactionHandlerTest.java

```diff
@@ -0,0 +1,137 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.impl.StorageDaoImpl;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TransactionHandlerTest {
+    public static final int POSITIVE_QUANTITY = 1;
+    public static final int ZERO_QUANTITY = 0;
+    public static final int NEGATIVE_QUANTITY = -1;
+    private StorageDao storageDao;
+
+    @Before
+    public void before() {
+        storageDao = new StorageDaoImpl();
+    }
+
+    @After
+    public void after() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    public void balanceOperationHandler_Ok() {
```

Comment:
<img width="372" alt="Screen Shot 2022-09-14 at 8 20 30 PM" src="https://user-images.githubusercontent.com/90692308/191308867-8f8428ad-c761-4d21-8051-9ffd145d8ddb.png">
The naming of your methods should look like this. Make it look like this everywhere